public/js/chat.js

```diff
@@ -12,12 +12,14 @@
   let endbtn = document.querySelector('#endbtn');
   let newbtn = document.querySelector('#newbtn');
   let cancel = document.querySelector('#cancel');
+  let end = document.querySelector('#end');
```

Comment:
check which is the correct end button and proceed. maybe if you emit disconnect event rather than clicking on the button, this wont be needed.